### PR TITLE
Revival of MadeAtPenn

### PR DESCRIPTION
**Pull request by adelq**
First attempt at taming the beast - main TODOs are to fix image uploads and use environment variables for configuration.

### CODE REVIEW

**Comment by armic** [comment]
@adelq can I help on this at all?

File: routes.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ var Tag = require('./models/projects').Tag;
 
 module.exports = function(app) {
   //controllers
+  // TODO: projects is undefined, and so this file is currently unusable
```
Comment (adelq) [inline routes.js]: I don't think `routes.js` is even used at all fwiw

**Comment by adelq** [comment]
Yup, I'm just fixing up some stuff, appreciate the help.

File: app.js [hunk 2]
```diff
@@ -42,20 +41,21 @@ app.get("/edit/:id", controller.edit);
...
 app.get("/login", controller.login);
-app.post("/check_login", controller.check_login);
 app.get("/logout", controller.logout);
 
+app.post("/update", controller.update_project);
+app.post("/approve", controller.approve_project);
+app.post("/upload", controller.upload);
+app.post("/create_account", controller.create_account);
+app.post("/post", controller.create);
+app.post("/check_login", controller.check_login);
```
Comment (adelq) [inline app.js:55]: Why are these routes set separately?

Comment (armic) [reply app.js:55]: @adelq just separating get and post requests for clarity.  Please feel free to revert if you prefer it the old way.

Comment (adelq) [reply app.js:55]: Ah OK, just noticed that. :+1:

**Comment by adelq** [comment]
Seems like MVP works, next step is to get a production environment going and deploy

**Comment by adelq** [comment]
Deployment is now live at http://madeatpenn.apps.pennlabs.org/faq, but the Mongo queries seem to be timing out, possibly because the packages are really outdated.

**Comment by armic** [comment]
@adelq if you want, we can basically do a rewrite of the whole thing.  There's not much here

**Comment by adelq** [comment]
There's no need, the project already works and doesn't merit a rewrite based on usage.

**Comment by armic** [comment]
okay, fair 👍

**Comment by adelq** [comment]
Alright, everything works! http://madeatpenn.apps.pennlabs.org/

**Pull request merged**